proto/forch_configuration.proto

```diff
@@ -165,6 +168,28 @@ message EventClientConfig {
  * Varz interface section in Forch configuration
  */
 message VarzInterfaceConfig {
-  // port number to curl for varz list
+  // port number for forch varz
   int32 varz_port = 1;
 }
+
+/*
+ * Target to scrape metrics from
+ */
+message ProxyServerConfig {
+  // port number for varz proxy
+  int32 proxy_port = 1;
+
+  // list of targets to scrape metrics from
+  repeated MetricTarget targets = 2;
```

Comment:
this should be a map of names that point at targets. It's nonsensical to have two names that are the same, and maps are strictly better than lists for sorting information unless you need to preserve ordering or allow duplicated.  See ProcessConfig for an example.  (Note that the value of the map should still be an object, even if it only has one item in it... as future-proofing)

Reply:
Addressed.